faiss/utils/utils.cpp

```diff
@@ -173,25 +191,6 @@ void reflection_ref (const float * u, float * x, size_t n, size_t d, size_t nu)
  * Some matrix manipulation functions
  ***************************************************************************/
 
-
```

Comment:
why did you move this away?

Reply:
Declaration was in `distances.h` (not included here, which led to link errors on Windows).